integrationTests/testInitializer.go

```diff
@@ -301,16 +324,114 @@ func MakeDisplayTable(nodes []*TestProcessorNode) string {
 }
 
 // PrintShardAccount outputs on console a shard account data contained
-func PrintShardAccount(accnt *state.Account) {
-	str := fmt.Sprintf("Address: %s\n", hex.EncodeToString(accnt.AddressContainer().Bytes()))
+func PrintShardAccount(accnt *state.Account, tag string) {
+	str := fmt.Sprintf("%s Address: %s\n", tag, base64.StdEncoding.EncodeToString(accnt.AddressContainer().Bytes()))
 	str += fmt.Sprintf("  Nonce: %d\n", accnt.Nonce)
-	str += fmt.Sprintf("  Balance: %d\n", accnt.Balance)
+	str += fmt.Sprintf("  Balance: %d\n", accnt.Balance.Uint64())
 	str += fmt.Sprintf("  Code hash: %s\n", base64.StdEncoding.EncodeToString(accnt.CodeHash))
 	str += fmt.Sprintf("  Root hash: %s\n", base64.StdEncoding.EncodeToString(accnt.RootHash))
 
 	fmt.Println(str)
 }
 
+// CreateDummyAddress creates a random byte array with fixed size
+func CreateDummyAddress() state.AddressContainer {
+	buff := make([]byte, sha256.Sha256{}.Size())
+	_, _ = rand.Reader.Read(buff)
+
+	return state.NewAddress(buff)
+}
+
+// CreateDummyHexAddress returns a string encoded in hex with the given size
+func CreateDummyHexAddress(chars int) string {
```

Comment:
suggestion to rename CreateRandomHexAddress

Reply:
Renamed.